openedx_certificates/renderers/elements.py

```diff
@@ -139,3 +139,13 @@ def draw_text(certificateGen, attributes, canvas, context):
 
 def draw_template_element(certificateGen, element, attributes, canvas, context=None):
     ELEMENT_OPTIONS[element](certificateGen, attributes, canvas, context)
+
+
+def draw_flair(certificateGen, flair, order, page, context=None):
```

Comment:
I think you may be talking about something else.
I'm referring to `draw_flair`; it's only ever called twice,
both invocations added in this PR, and each one
passes `context`. 

There are other methods where `context` isn't always passed,
but I don't think that's the case with `draw_flair`.